### ISSUE TITLE
IOS Navigation Testing Feedback

### ISSUE THREAD

**Issue by shrutikamati** [sriramrathinavelu/attendanceSWE2015#26]
Changes required for the app based on the version provided on 10/28/2015

1:First screen: Whenever I open the app it is showing dashboard. It should show splash screen which will be displayed for few seconds and must contain the logo with white background.

2:Splash screen: For splash screen there should not be any options like manage courses, dashboard and more. These options should come When the sign in or registration is successful.

3:Log in page: When I give some mail id and password and click on sign in it is showing some pop up notification with the details entered. And it is showing password also. I feel validation pop up notification is not required. As discussed email id must be mail id provided by ITU(domain validation).Password must contain 7 characters. Now it is reading any number of characters.

4:Registration page: In registration screen, I can’t see the password & email attribute when I try to provide inputs. And same as log in page the pop up notification is showing all the inputs including password. Which I think is not necessary. 

5:Manage course: The delete and add buttons are not working. According to Balsamiq when we slide to left the delete option should come. There should me limitation for adding the courses. Because according to university a student can take 4 courses per trimester. 

6:Course detail screen: Current date & time is showing instead of it must show class timings. For the day of the week it is showing Sunday and for Duration it is showing October 31 to October 31. According to PRD and Balsamiq Whenever we add a weekend course the set of dates on which the class is there must show. And when we add weekday the duration can be xx date to yy date as it is there in the screen.

7:Generate Reports: The toast is missing. This will say the email is sent to you with class attendance report.

8:Manual attendance: This screen is empty. Class Date and Student ID and submit button is missing.

**Comment by shrutikamati** [comment]
Is this okay or I need to create a separate issue for every screen?

**Comment by winddweb** [comment]
### First screen
- [x] add splash screen
- [x] show only login page 

### Log in page:
- [x] add email validation
- [x] add password validation, 7 chars minimum

**to be discussed**
login form is unified for professor and students
if so, if the email ends with @itu.edu, we consider it's a professor account

### Registration page
- [x] password & email fields blocked by keyboard _on small screen_

### Manage course
- [ ] add **delete** function
- [ ] add **add** function
- [x] slide to show delete button
- [ ] add limitation of max course number to 4

**Comment by winddweb** [comment]
@SAT15 I found that the checklist is working in ISSUEs, but not in WIKI pages.

**Comment by winddweb** [comment]
## Update

### what's left

### Log in page:
- [x] add email validation
- [x] add password validation, 7 chars minimum

### Manage course
- [ ] add **delete** function
- [ ] add **add** function
- [ ] add limitation of max course number to 4

**Comment by shrutikamati** [comment]
How did you create these check boxes?

**Comment by winddweb** [comment]
```
- [x] this is a complete item
- [ ] this is an incomplete item
```

**Comment by shrutikamati** [comment]
add - [*] this is a complete item

**Comment by shrutikamati** [comment]
- [ ] add

**Comment by shrutikamati** [comment]
Got it:)

**Comment by winddweb** [comment]
I think what's left here should be `Task`s not `Issue` or `defect`, so let's close this one
